specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json

```diff
@@ -3403,6 +3464,112 @@
         }
       }
     },
+    "ResourceTypeSkuCollection": {
+      "description": "Collection of supported Skus of Azure Spring Cloud resources",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of supported Skus",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceTypeSku"
+          }
+        }
+      }
+    },
+    "ResourceTypeSku": {
```

Comment:
>"ResourceTypeSku" [](start = 4, length = 17)

this is the private contract definition (between your RP and ARM), not the public definition.  See specification\datamigration\resource-manager\Microsoft.DataMigration\preview\2017-11-15-preview\definitions\ResourceSkus.json for an example of the public definition

Reply:
@pilor , thanks. Added ResourceSku in swagger. Please review it.